mahalia/storage.mk

```diff
@@ -0,0 +1,96 @@
+# deb package storage management Makefile, copied from openMHA and adapted.
+
+# Debian package management by Jenkins:
+# New Debian Packages are stored in our storage for debian repositories.
+# The storage is cleaned of old packages depending on the current branch.
+#
+# Glossary:
+#
+# supply:
+# The packages stored here have just been built. Location is
+# project-specific. For openMHA, it is ./mha/tools/packaging/deb/hoertech/$SYSTEM/
+# regardless of $BRANCH_NAME.
+# The packages here may be new or they may be rebuilds of
+# existing versions (e.g. if someone clicks build-now while
+# there is no new revision). Rebuilds of existing versions
+# will not be used, but will cause an update of the timestamp of the respective
+# files in storage.  New packages are copied to storage
+#
+# storage:
+# The packages here are kept persistently across builds.
+# Old packets (timestamp older than some threshold) will be deleted for
+# branch development.
+# Storage location:
+# /var/lib/jenkins/packages-for-aptly/STORAGE/$PROJECT/$BRANCH_NAME/$SYSTEM/
+# on the host, which is mounted to /STORAGE/$PROJECT/$BRANCH_NAME/$SYSTEM/
+# in the container.
+#
+#
+# $PROJECT:
+# openMHA, liblsl, liblsl-matlab, tascarpro, more may be added
+#
+# $BRANCH_NAME:
+# master, uploaded to apt.hoertech.de, and development, uploaded to
+# aptdev.hoertech.de. BRANCH_NAME is normally set by Jenkins.
+#
+# HOWEVER, for this project (mahalia-kernel), we cannot use branch "master" in
+# git because "master" refers to Torvalds' version of the kernel. But since we
+# will probably never build different kernels for development and releases, we
+# can just override BRANCH_NAME here:
+BRANCH_NAME=master
+export BRANCH_NAME
+#
+# SYSTEMs:
+# xenial, bionic, focal, buster, etc. Available SYSTEMs that
+# contain packages are detected automatically with $(wildcard)
+
+# There will by $SYSTEM subdirectories below this directory.
+# These subdirectories then contain the package files.
+SUPPLY_DIR = deb
```

Comment:
This Makefile creates

STORAGE/mahalia-kernel/master/deb/bionic/mahalia-kernel_4.14.94.mha.4.12.0-1_armhf.deb

The intermediate directory "deb" should not be there.